tools/interop-node/feeder/block_feeder.go

```diff
@@ -118,6 +118,7 @@ func (feeder *BlockFeeder) HandlePrevote(ctx context.Context, height int64) erro
 
 	if err := feeder.sendPrevote(ctx, blockDataStr, salt); err != nil {
 		feeder.logger.Error(fmt.Sprintf("failed to send prevote: %v", err))
```

Comment:
Then we don't need to print a log at this line because the error will be caught and logged in the main loop.